src/Tasks/ResolveAssemblyReferences/Server/RarController.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.IO.Pipes;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Tasks.ResolveAssemblyReferences.Contract;
+using Microsoft.Build.Tasks.ResolveAssemblyReferences.Services;
+using Microsoft.VisualStudio.Threading;
+
+using StreamJsonRpc;
+
+#nullable enable
+namespace Microsoft.Build.Tasks.ResolveAssemblyReferences.Server
+{
+    public sealed class RarController
+    {
+        private const int PipeBufferSize = 131072;
```

Comment:
Where did this number come from?

Reply:
Removed, it was there from previous copying of code, it is default size of buffer for named pipes in OutOfProcNode